Packages/vcs/Lib/fillarea.py

```diff
@@ -336,6 +355,7 @@ def __init__(self, Tf_name=None, Tf_name_src='default'):
             self._style = ['solid', ]
             self._index = [1, ]
             self._color = [1, ]
+            self._opacity = []
```

Comment:
Nope. The default is different in different cases.
For solids and patterns, default opacity value is 0 and for hatches it is 100.